java/java.hints/src/org/netbeans/modules/java/hints/jdk/ConvertToPatternInstanceOf.java

```diff
@@ -89,7 +90,9 @@ public Void visitTypeCast(TypeCastTree node, Void p) {
                 TypeMirror typeITM = ctx.getInfo().getTrees().getTypeMirror(typeI);
                 List<String> varNameCandidates = org.netbeans.modules.editor.java.Utilities.varNamesSuggestions(typeITM, ElementKind.LOCAL_VARIABLE, EnumSet.noneOf(Modifier.class), null, null, ctx.getInfo().getTypes(), ctx.getInfo().getElements(), Collections.emptyList(), CodeStyle.getDefault(ctx.getInfo().getFileObject()));
                 String varName = Utilities.makeNameUnique(ctx.getInfo(), ctx.getInfo().getTrees().getScope(ctx.getPath()), varNameCandidates.get(0));
-                Fix fix = new FixImpl(ctx.getInfo(), ctx.getPath(), varName, false, convertPath).toEditorFix();
+                IfTree it = (IfTree) ctx.getPath().getLeaf();
```

Comment:
FWIW, I'd be fine if this hint only supported JDK 16+, and the final version of pattern matching in instanceof - hopefully that would make things a little easier. But not that strong opinion.

Reply:
I believe its better to keep this at least for this release. As many people using below JDK-16 version. Thanks.